src/Database/Reader.php

```diff
@@ -110,6 +110,26 @@ public function anonymousIp($ipAddress)
     }
 
     /**
+     * This method returns a GeoIP2 ASN model.
+     *
+     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     *
+     * @return \GeoIp2\Model\Asn
+     *
+     * @throws \GeoIp2\Exception\AddressNotFoundException if the address is
```

Comment:
@oalders Should there be tests that assert that these exceptions actually get thrown? `flatModelFor()` is a private method and none of the exceptions thrown by it are tested for, as far as I can tell. If the underlying exceptions change, it could break someone's app who is trying to specifically catch and handle these exceptions.